src/hooks/useApiChainName.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import { ApiPromise } from '@polkadot/api';
+import { useEffect, useState } from 'react';
+import logger from '../util/logger';
+import { getChainName } from '../util/getConfigs';
+import { ConnectionChainInformation } from '../types/sourceTargetTypes';
+import { BASE_CUSTOM_TYPES } from '../constants';
```

Comment:
I'm a bit confused, why do we still need `BASE_CUSTOM_TYPES` and `useApiChainName` hook at all? Since that with your decision to put custom types URLs to ENV variables (which is not something that I suggested in other PR, but I do like (cause it's even more configurable)) we can create API connection right away with proper types?